playbooks/roles/ocp-descheduler/tasks/main.yaml

```diff
@@ -0,0 +1,177 @@
+            sourceNamespace: openshift-marketplace
+  when: descheduler_operator_deploy
+
+- name: Verify the operator installation
+  block:
+    - name: Check if the cluster service version has status as Succeeded
+      shell: "oc get csv --all-namespaces --no-headers | grep clusterkubedescheduleroperator | awk '{ if (($2 ~ /clusterkubedescheduleroperator/) && $NF  == \"Succeeded\") print $2 }'| wc -l"
+      register: descheduler_operators_csv
+      until: descheduler_operators_csv.stdout|int != 0
+      retries: 15
+      delay: 60
+
+    - name: Check if pods are running
+      shell: "oc get pods --all-namespaces --no-headers | grep descheduler | awk '{ if ($4 == \"Running\") print $2 }' | wc -l"
+      register: descheduler_pods
+      until: descheduler_pods.stdout|int != 0
+      retries: 15
+      delay: 60
+
+# Descheduler e2e run block
+- name: Running e2e test suite in openshift-tests-private for descheduler
+  block:
+    - name: Create a directory for clonning the repo
+      file:
+        state: directory
+        path: "{{ descheduler_workdir }}/e2e-test-result/"
+        mode: 0755
+
+    - name: Install the developer tools
+      yum:
+        name: '@Development tools' 
+        state: present
+
+    - name: Get the golang version
+      shell: go version | awk -F'.' '{ print $2 }'
+      register: go_minor_version
+    
+    - name: Install the latest version of go
+      include_role: 
+        name: golang-installation
+      vars:
+        go_tarball: "{{ descheduler_go_tarball }}"
+        golang_installation_path: "/usr/local"
+      when: go_minor_version.stdout|int < 18
+
+    - name: Clone the openshift-tests-private repo
+      git:
+        repo: "https://{{ descheduler_github_username }}:{{ descheduler_github_access_token }}@github.com{{ descheduler_e2e_repo | urlsplit('path') }}"
+        dest: "{{ descheduler_e2e_dir }}"
+        version: "{{ descheduler_e2e_github_branch }}"
+      when: descheduler_e2e_repo != "" or descheduler_e2e_repo != None
+
+    - name: Replace the catalogsource name in the kube_descheduler_operator test module
+      ansible.builtin.replace:
+        path: "{{ descheduler_e2e_dir }}/test/extended/workloads/kube_descheduler_operator.go"
+        regexp: 'opsrcName\:\s*\"[a-zA-Z0-9\-]*\"'
+        replace: "opsrcName:  \"{{ descheduler_catalogsource_name }}\""
+
+    - name: Replace the descheduler operator image build with the custom one in the kube_descheduler_operator test module
+      ansible.builtin.replace:
+        path: "{{ descheduler_e2e_dir }}/test/extended/workloads/kube_descheduler_operator.go"
+        regexp: 'imageInfo\:\s*\"[a-z0-9\.\/\:\-]*\"'
+        replace: "imageInfo:  \"{{ descheduler_catalogsource_image }}\""
+
+    - name: Compile the go modules using make command
+      make:
+        chdir: "{{ descheduler_e2e_dir }}"
+        target: build
+      
+    - name: Create a bin executable extended-platorm-tests
```

Comment:
looks like this is not required.

Reply:
Removed this step from the tasks main playbook.